blackbox/docs/sql/statements/copy-from.txt

```diff
@@ -35,15 +35,31 @@ Description
 import.
 
 The nodes in the cluster will attempt to read the files available at the URI
-and import the data. These files have to be UTF-8 encoded and contain a single
+and import the data.
+
+Supported Formats
+-----------------
+CrateDB accepts both JSON and CSV inputs. This can be specified using either the
+relevant file extension, or by providing the file type as an option (see ``WITH``
+below), which may be useful if the files do not have a `.json` or `.csv` file extension.
+In the case that a file does not have a relevant file extension and format option has not
+been provided, then by default the file with be processed as JSON.
+
+Files have to be UTF-8 encoded, and in the case of JSON, contain a single
 JSON object per line. Any keys in the object will be added as columns,
-regardless of the previously defined table. Empty lines are simply sikpped.
+regardless of the previously defined table. Empty lines are simply skipped.
 
 Example JSON data::
 
     {"id": 1, "quote": "Don't panic"}
     {"id": 2, "quote": "Ford, you're turning into a penguin. Stop it."}
 
+Example CSV data::
+
+    id,quote
```

Comment:
I wonder if this is enough for users to grasp that the header is mandatory? (It is I assume?) Maybe we should point this out explicitly in the paragraph above

Reply:
👍 6f87a63